x/concentrated-liquidity/genesis.go

```diff
@@ -184,15 +196,35 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *genesis.GenesisState {
 		if err != nil {
 			panic(err)
 		}
+
 		feeAccumPositionRecord, err := feeAccumObject.GetPosition(feePositionKey)
 		if err != nil {
 			panic(err)
 		}
 
+		// Retrieve uptime incentive accumulator state for position
+		uptimePositionKey := string(types.KeyPositionId(position.PositionId))
+		uptimeAccumObject, err := k.GetUptimeAccumulators(ctx, position.PoolId)
+		if err != nil {
+			panic(err)
+		}
+
+		var uptimeAccumRecords []accum.Record
+
+		for _, uptimeAccum := range uptimeAccumObject {
+			uptimeAccumPositionRecord, err := uptimeAccum.GetPosition(uptimePositionKey)
+			if err != nil {
+				panic(err)
+			}
+
+			uptimeAccumRecords = append(uptimeAccumRecords, uptimeAccumPositionRecord)
+		}
```

Comment:
Notice how here we are getting a position for every uptime.

Then in the import logic, for every exported position we initialize an uptime position which isn't correct.

I think the tests are passing because of being too simple where all records are the same. Please add test cases with different records, I think they might expose what I'm trying to point at